easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-1.2.2-GCC-13.2.0.eb

```diff
@@ -0,0 +1,44 @@
+# EasyBuild easyconfig
+#
+# Contributed from Fred Hutchinson Cancer Research Center, Seattle WA, US 
+# John Dey jfdey@fredhutch.org
+#
+easyblock = 'CMakeMake'
+
+name = 'RAxML-NG'
+version = '1.2.2'
+
+homepage = 'https://github.com/amkozlov/raxml-ng'
+description = """RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)
+ optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree
+ Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+sources = [{
+    'filename': '%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/amkozlov',
+        'repo_name': '%(namelower)s',
+        'tag': '%(version)s',
+        'recursive': True,
+        'keep_git_dir': True,
+    }
+}]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('GMP', '6.3.0'),
+]
+
```

Comment:
I think you meant to add
```
configopts = '-DUSE_GMP=ON'
```

Reply:
Thanks for spotting that. I have also enabled the test, just to be sure.